artifacts/rhdh-plugins-reference/argocd/argocd-plugin-admin.adoc

```diff
@@ -70,6 +70,143 @@ global:
         disabled: false
 ----
 
+== (Optional) Enabling Argo CD Rollouts feature
+
+The Argo CD Rollouts feature enhances Kubernetes by providing advanced deployment strategies, such as blue-green and canary deployments, for your applications. When integrated into the backstage Kubernetes plugin, it allows developers and operations team to visualize and manage these Rollouts seamlessly within the Backstage interface.
+
+.Prerequisites
+
+* *Backstage Kubernetes plugin installed:* Verify that the @backstage/plugin-kubernetes-backend is installed and configured. Follow the installation guide and configuration guide if needed.
```

Comment:
> Follow the installation guide and configuration guide if needed.

installation and configuration should be links, use the below links to redirect users to that documentation.

installation link :  https://backstage.io/docs/features/kubernetes/installation/
configuration link: https://backstage.io/docs/features/kubernetes/installation/